ui/css/perf.css

```diff
@@ -1,5 +1,11 @@
-body {
+html {
   overflow: auto;
+  height: auto;
+}
+
+body {
+  overflow: visible;
+  height: auto;
```

Comment:
Fun fact: If you want to use window.scrollTo methods, you need to have the scrolling set to `overflow: auto` for html, not body (Perfherder was inheriting `overflow: hidden` from global CSS settings), otherwise it won't work because the scrolling is on the body.